MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,587 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI  ProgressBar | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin SfProgressBar to Syncfusion .NET MAUI SfProgressBar control and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfProgressBar to .NET MAUI SfProgressBar
+
+To make the migration from the [Xamarin SfProgressBar](https://www.syncfusion.com/xamarin-ui-controls/xamarin-progressbar) to `.NET MAUI SfProgressBar` easier, most of the APIs from the Xamarin SfProgressBar were kept in the.NET MAUI SfProgressBar. However, to maintain the consistency of API naming in the .NET MAUI SfProgressBar, some of the APIs have been renamed. Please find the difference in the following topics.
```

Comment:
To make the migration from the [Xamarin SfProgressBar](https://www.syncfusion.com/xamarin-ui-controls/xamarin-progressbar) to `.NET MAUI SfProgressBar` easier, most of the APIs from the Xamarin SfProgressBar were kept in the.NET MAUI SfProgressBar. However, to maintain the consistency of API naming in the .NET MAUI SfProgressBar, some of the APIs have been renamed. Please find the difference in the following topics:

Reply:
not included colon